tools/linter.py

```diff
@@ -74,10 +38,6 @@ def run_lint(self, uncommitted):
 
 if __name__ == '__main__':
     parser = ArgumentParser()
-    parser.add_argument("--branch", type=str, default='main',
```

Comment:
Ah,  here it is. The branch argument is no longer needed?

Reply:
I don't think it's need any more, because ruff applies to the whole code base, not only diffs.